code/modules/clothing/masks/breath.dm

```diff
@@ -186,3 +192,35 @@
 			squad_color = squad_name
 	icon_state = replacetext("[initial(icon_state)][pulled ? "_down" : ""]", "%SQUAD%", squad_color)
 	item_state = replacetext("[initial(item_state)][pulled ? "_down" : ""]", "%SQUAD%", squad_color)
+
+
+/obj/item/clothing/mask/rebreather/tornscarf/classic
+	name = "tactical scarf"
+	desc = "A tactical scarf used to keep warm in the cold."
+	icon_state = "torn_scarf_classic"
+	item_state = "torn_scarf_classic"
+	flags_inventory = COVERMOUTH|ALLOWREBREATH|ALLOWCPR
+	flags_inv_hide = HIDEFACE|HIDELOWHAIR
+	flags_cold_protection = BODY_FLAG_HEAD
+	min_cold_protection_temperature = ICE_PLANET_min_cold_protection_temperature
```

Comment:
none of this stuff will be inherited because you have made it tornscarf/classic and the types below are not subtypes of /classic they are subtypes of /tornscarf